build.sh

```diff
@@ -225,17 +225,16 @@ if [ "$clangTidy" -eq 1 ] && [ `command -v clang-tidy` ]; then
         #clang-tidy -p $compile_json $cpp -checks='*' -header-filter='.*'
 
         # Don't check header, don't check google/llvm coding conventions
-        echo "$count/$total"
```

Comment:
It would be nice to fix the code. Or add a progress bar, I don't know if there is any standard stuff.